pkgs/by-name/fl/flashprog/package.nix

```diff
@@ -12,14 +12,14 @@
 , withGpio ? stdenv.isLinux
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
```

Comment:
Can you please change this to 

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

`rec` is discouraged, and I don't want to include it more.
Otherwise LGTM.

Reply:
Thanks for pointing that out. Done.